constants.js

```diff
@@ -1,4 +1,4 @@
 
 
 export const env = window.location.hostname === 'localhost' ||  window.location.hostname === '127.0.0.1' ? 'dev' : 'prod';
-export const apiOrigin = env === 'dev' ? 'http://127.0.0.1:80/ipfs' : 'https://ipfs.exokit.org/ipfs';
\ No newline at end of file
+export const apiHost = env === 'dev' ? 'http://127.0.0.1' : 'https://ipfs.exokit.org';
```

Comment:
I'd rather we put the full url in a variable here. It wouldn't technically be a "host" but it might change, so it's good to abstract it in some variable.

Reply:
hmm, yes we could make a `routes.js` file that is basically a list of all our endpoints in exportable constants format... could refactor some of the `if else` routing going on in `index.js at the bottom. Good idea.